consumer/start-consumer.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# set image and container names
+IMAGE_NAME="kafka-consumer"
+CONTAINER_NAME="kafka-consumer-container"
+FORCE_REBUILD=false
+FORCE_RESTART=false
+
+# Parse arguments
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    --force-rebuild)
+      FORCE_REBUILD=true
+      shift # past argument
+      ;;
+    --force-restart)
+      FORCE_RESTART=true
+      shift # past argument
+      ;;
+    *)
+      shift # past unrecognized argument
+      ;;
+  esac
+done
+
+# Check if the Docker image already exists
+if [[ "$(docker images -q $IMAGE_NAME 2> /dev/null)" == "" || "$FORCE_REBUILD" = true ]]; then
```

Comment:
why or?

Reply:
Or because the --force-rebuild option should override the presence of an already existing image. It's intentional